packages/components/src/components/SquiggleViewer/utils.ts

```diff
@@ -122,3 +124,24 @@ export function useGetSubvalueByPath() {
     return value;
   };
 }
+
+const tagsDefaultCollapsed = new Set(["Bool", "Number", "Void", "Input"]);
+
+export function hasExtraContentToShow(v: SqValueWithContext): boolean {
+  return !(
+    tagsDefaultCollapsed.has(v.tag) ||
+    (v.tag === "String" && v.value.length <= SHORT_STRING_LENGTH)
```

Comment:
I think this should also check if the value has a comment; otherwise the comment would only be available through the tooltip, and comments can potentially be too long for tooltips:

https://squiggle-website-git-frontend-cle-1735e1-quantified-uncertainty.vercel.app/playground?v=dev#code=eNo1kLtumzEMhV%2FlwNmMwJmyBOiUdmvHZPLCSPRvAhKlSKThIMi7l4rbTQLJc%2Fk%2Bd5lP5MWeW%2Bbd0%2B5hvz%2FqHV55fKA03ZBbmjZEt6PG4A5%2FZDsbRFPxzJh84UEF09%2FOTJnHXGu%2F2%2BAK6dNr3Jc2MMVAle0eqenkZGw%2BQFm6zBTi4CIxnJzjACw%2Ba8swrj2Ow0yyZFeDGwq9hTzYbtKMSpsSqMi70wEvBlapoY0q63GJL9V7vLtMaIsynsFXHkmMTJrCS6Ga2k15LcmU5fQtKT2WwRTBa2RqtwJhZQf8XJLkxpDhkeTWVRSD%2B%2BAzawCRBQuXVryHHUecaAqek5GklP%2BEopDj5JuQQVcgdBrx8XHAr2vibuwLYzBoKRGn2EveJZOti2jRR5PMuiguUmGavHRavdFOJ0lCyDx5rGltZcWgBUgCx%2FzH1evhqPuHo17xA4%2B7r79Em8cQ

<img width="1011" alt="Screenshot 2023-12-22 at 18 11 47" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/0c57b475-5df9-4e3a-8455-3f7dc1fda7ee">

Reply:
Good point